### DISCUSSION TITLE
想问问大家，如何用LogicFlow实现泳道图嘛

### DISCUSSION THREAD

**Discussion by wumail** [didi/LogicFlow#1128] [Ideas]
最近有需求需要实现泳道图，想用LogicFlow实现来，感觉是可以的，有什么比较好实践嘛？

**Comment by boyongjiong** [comment]
你看下这个示例能不能满足你的要求 https://github.com/Logic-Flow/docs/tree/master/demo/pool

预览可以看这个： https://docs.logic-flow.cn/demo/dist/pool/
